src/setup.py

```diff
@@ -1,2 +1,5 @@
 from setuptools import setup
-setup()
+setup(
```

Comment:
@vickywane we are using the [setup.cfg](https://github.com/ambianic/ambianic-edge/blob/master/src/setup.cfg) mechanism for declaring metadata including version. setup.py reads it automatically. No need to set name and version in setup.py. That breaks the way the semantic release script is tied to version updates.